src/containers/Simularium/index.tsx

```diff
@@ -217,10 +218,11 @@ class App extends React.Component<AppProps, AppState> {
                             setViewerStatus={setViewerStatus}
                             setError={setError}
                         />
-                        <SideBar
-                            onCollapse={this.onPanelCollapse}
-                            type="left"
-                        >
+                        <ConversionFormOverlay
+                            className="conversionform"
+                            isLoading={false}
```

Comment:
this should be `isImporting` and should be using the `VIEWER_IMPORTING` value in state

Reply:
or it could just be called `active` too for simplicity

Reply:
yes, i left this static for ease during development, will change